<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1543674695300" stop="1543674698045">
	<name>Adding customer information</name>
	<title>Adding customer information</title>
	<test-cases>
		<test-case start="1543674695300" stop="1543674697091" status="failed">
			<name>Adding customer</name>
			<title>Adding customer</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="bf62e490-e33b-4999-88e7-b0df387c2bf2-attachment.png" type="image/png" size="4687"/>
			</attachments>
			<failure>
				<message>Failed: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;</message>
				<stack-trace>Error: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;
    at runWaitForAngularScript.then (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\browser.js:463:23)
    at ManagedPromise.invokeCallback_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at addCustomer.gotoAddCustomer (C:\Users\Prashant\BankingProjectWS\BankingProject\pages\addCustomer.js:17:16)
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:12:11)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it(&quot;Adding customer&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
From asynchronous test: 
Error
    at C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:11:2
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:20:20
    at Suite.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:9:2)
    at addSpecsToSuite (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:8:1)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1543674697091" stop="1543674697404" status="failed">
			<name>Adding customer</name>
			<title>Adding customer</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="e45c0142-0283-4fe4-bfb6-bd6622f5e5ab-attachment.png" type="image/png" size="4687"/>
			</attachments>
			<failure>
				<message>Failed: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;</message>
				<stack-trace>Error: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;
    at runWaitForAngularScript.then (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\browser.js:463:23)
    at ManagedPromise.invokeCallback_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at addCustomer.gotoAddCustomer (C:\Users\Prashant\BankingProjectWS\BankingProject\pages\addCustomer.js:17:16)
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:12:11)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it(&quot;Adding customer&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:11:2
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:20:20
    at Suite.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:9:2)
    at addSpecsToSuite (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:8:1)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1543674697404" stop="1543674697728" status="failed">
			<name>Adding customer</name>
			<title>Adding customer</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="62e96c09-ebbe-4547-9a29-db06d976dec5-attachment.png" type="image/png" size="4687"/>
			</attachments>
			<failure>
				<message>Failed: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;</message>
				<stack-trace>Error: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;
    at runWaitForAngularScript.then (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\browser.js:463:23)
    at ManagedPromise.invokeCallback_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at addCustomer.gotoAddCustomer (C:\Users\Prashant\BankingProjectWS\BankingProject\pages\addCustomer.js:17:16)
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:12:11)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it(&quot;Adding customer&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:11:2
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:20:20
    at Suite.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:9:2)
    at addSpecsToSuite (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:8:1)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1543674697728" stop="1543674698045" status="failed">
			<name>Adding customer</name>
			<title>Adding customer</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="9d2bc702-e9fb-4ea3-8c11-4a4f78ad95c7-attachment.png" type="image/png" size="4687"/>
			</attachments>
			<failure>
				<message>Failed: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;</message>
				<stack-trace>Error: Error while waiting for Protractor to sync with the page: &quot;both angularJS testability and angular testability are undefined.  This could be either because this is a non-angular page or because your test involves client-side navigation, which can interfere with Protractor&apos;s bootstrapping.  See http://git.io/v4gXM for details&quot;
    at runWaitForAngularScript.then (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\browser.js:463:23)
    at ManagedPromise.invokeCallback_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
    at TaskQueue.executeNext_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
    at asyncRun (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
    at addCustomer.gotoAddCustomer (C:\Users\Prashant\BankingProjectWS\BankingProject\pages\addCustomer.js:17:16)
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:12:11)
    at C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
    at new ManagedPromise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
    at ControlFlow.promise (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
    at schedulerExecute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
    at TaskQueue.execute_ (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
From: Task: Run it(&quot;Adding customer&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at QueueRunner.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
    at Spec.queueRunnerFactory (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
    at Spec.execute (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
    at UserContext.fn (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
    at attempt (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
    at QueueRunner.run (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
    at runNext (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4257:20)
From asynchronous test: 
Error
    at C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:11:2
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at C:\Users\Prashant\BankingProjectWS\BankingProject\node_modules\jasmine-data-provider\src\index.js:20:20
    at Suite.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:9:2)
    at addSpecsToSuite (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
    at Env.describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
    at describe (C:\Users\Prashant\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (C:\Users\Prashant\BankingProjectWS\BankingProject\test_specs\addCustomerInfo.js:8:1)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>